src/modules/powerrename/dll/dllmain.cpp

```diff
@@ -224,6 +224,9 @@ class PowerRenameModule : public PowertoyModuleIface
         return 0;
     }
 
+    virtual void register_system_menu_helper(PowertoySystemMenuIface* helper) override { }
+    virtual void signal_system_menu_action  (const wchar_t* name) override { }

```

Comment:
Remove the extra space between `signal_system_menu_action` and the parameter list.